source/src/cip/cipassembly.c

```diff
@@ -31,7 +31,7 @@ EipStatus SetAssemblyAttributeSingle(CipInstance *const instance,
  */
 CipClass *CreateAssemblyClass(void) {
   /* create the CIP Assembly object with zero instances */
-  CipClass *assembly_class = CreateCipClass(kCipAssemblyClassCode, 0, /* # class attributes*/
+  CipClass *assembly_class = CreateCipClass(kCipAssemblyClassId, 0, /* # class attributes*/
```

Comment:
The official name is Class Code (see Vol. 1 4-3), so this should be reverted

Reply:
To change this back is ok for me.

The reason for me to change the names of the constants to contain a "ClassId" postfix was that in the CipInstance the class code is memorized in the "class_id" member. Therefore in some parts of the code there was a comparison like "instance->cip_class->class_id == kCipAssemblyClassCode". I found it counter intuitive to compare a "class_id" with a "k*ClassCode" constant.

To be completely in line with the spec the member variable should then be named "class_code" and the constants also "k*ClassCode".

Reply:
Oh, thanks. You are right this is then counterintuitive. so I will also change this.

My reasoning for this is, bacl when I took over development a lot of variabables and constants were named differently than in the CIP spec. which made it very hard to understand the code.